lms/static/js/groups/views/cohort_editor.js

```diff
@@ -5,10 +5,24 @@ var edx = edx || {};
 
     edx.groups = edx.groups || {};
 
+    edx.groups.reloadPage = function() {
+        location.reload(true);
+    };
+
+    edx.groups.setActionButtonsVisibility = function(showSettings) {
+        var formActions = '.cohort-management-details .form-actions';
+        if (showSettings) {
```

Comment:
I'm not clear what the point of this method is (especially given the current implementation with the reload). You are disabling the Save and Cancel buttons I guess, but only when the form has already completed saving. Why? If you want to prevent the user from submitting multiples times, the buttons should be disabled in the click handler before saveForm is called, and then re-enabled in both done and fail (or an event that is always triggered, regardless of success).